gulpfile.babel.js

```diff
@@ -167,5 +169,26 @@ gulp.task('test', [
 // Run unit tests
 gulp.task('lint', [
   'lint:styles',
-  'lint:scripts'
+  'lint:scripts',
+  'lint:json'
 ])
+
+gulp.task('lint:json', () => {
+  gulp.src(['./app/data/*.json'])
+    .pipe(prettify())
+    .pipe(diff())
+    .pipe(diff.reporter({
+      quiet: false,
+      fail: true
+    }))
+})
+
+gulp.task('format', () => {
```

Comment:
Just because it uses https://www.npmjs.com/package/js-beautify, I suppose if the only thing it is going to do is reformat the JSON, then format is fine.